cmd/spiffe-helper/config/config.go

```diff
@@ -24,6 +28,10 @@ type Config struct {
 	CmdArgsDeprecated                  string `hcl:"cmdArgs"`
 	CertDir                            string `hcl:"cert_dir"`
 	CertDirDeprecated                  string `hcl:"certDir"`
+	CertFileMode                       *int   `hcl:"cert_file_mode"`
+	KeyFileMode                        *int   `hcl:"key_file_mode"`
+	JwtBundleFileMode                  *int   `hcl:"jwt_bundle_file_mode"`
+	JwtSvidFileMode                    *int   `hcl:"jwt_svid_file_mode"`
```

Comment:
```suggestion
	JWTBundleFileMode                  *int   `hcl:"jwt_bundle_file_mode"`
	JWTSvidFileMode                    *int   `hcl:"jwt_svid_file_mode"`
```

Reply:
Also why use pointers here?

Reply:
I did that so they could be nil. Don't I have to do that to make it not required in the config?

Reply:
No, you can have it be a regular type and just check if its 0 and the assign the default. And probably error out if its < 0

Reply:
Updated.